projects/storefrontlib/src/cms-components/anonymous-consent-management/anonymous-consent-launch-dialog.service.ts

```diff
@@ -0,0 +1,37 @@
+import { ElementRef, Injectable, ViewContainerRef } from '@angular/core';
+import { combineLatest, Observable } from 'rxjs';
+import { filter, map, tap } from 'rxjs/operators';
+import {
+  LaunchDialogService,
+  LAUNCH_CALLER,
+} from '../../layout/launch-dialog/index';
+
+@Injectable({ providedIn: 'root' })
+export class AnonymousConsentLaunchDialogService {
+  constructor(protected launchDialogService: LaunchDialogService) {}
+
+  openDialog(
+    openElement?: ElementRef,
+    vcr?: ViewContainerRef
+  ): Observable<any> {
```

Comment:
Maybe indicate that the method can return `undefined` with `Observable<any> | undefined`?